lang/ucblogo/Portfile

```diff
@@ -1,38 +1,26 @@
-PortSystem 1.0
-
-name             ucblogo
-version          6.2.2
-revision         0
-categories       lang
-maintainers      nomaintainer
-platforms        darwin
-description      Berkeley Logo
-long_description \
-    Berkeley Logo is a free Logo interpreter.
-homepage         https://people.eecs.berkeley.edu/~bh/logo.html
-
+PortSystem          1.0
```

Comment:
After your reformatting, the Portfile now conforms to our standard modeline, so you should add it.
```suggestion
# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4

PortSystem          1.0
```
Normally reformatting should be done in a separate commit from functional changes, but since you're changing most of the lines anyway it's probably fine to combine it here.

Reply:
Hi @ryandesign, 

I have added this to the portfile.

Thanks,

-ranga